packages/requests/src/lib/requests-result.ts

```diff
@@ -198,13 +203,28 @@ export function trackRequestResult<TData>(
             ? options.preventConcurrentRequest
             : true;
 
+        const cacheResponseData = options?.cacheResponseData;
+
         if (
           !options?.skipCache &&
           (result.isSuccess ||
             (preventConcurrentRequest && result.fetchStatus === 'fetching')) &&
           !stale
         ) {
-          return EMPTY;
+          return getRequestResult(key).pipe(
+            switchMap((requestResult) => {
+              if (requestResult.isError) {
```

Comment:
What's the use case for isError? You'll get here only if it's success

Reply:
If `preventConcurrentRequest` is truthy this code can be reached for requests which are `fetching`, those requests can succeed or error

Reply:
So we need this chunk only in this case